pavelib/paver_tests/utils.py

```diff
@@ -0,0 +1,60 @@
+"""Unit tests for the Paver server tasks."""
+
+import os
+from paver import tasks
+from unittest import TestCase
+
+
+class PaverTestCase(TestCase):
+    """
+    Base class for Paver test cases.
+    """
+    def setUp(self):
+        super(PaverTestCase, self).setUp()
+
+        # Show full length diffs upon test failure
+        self.maxDiff = None  # pylint: disable=invalid-name
+
+        # Create a mock Paver environment
+        tasks.environment = MockEnvironment()
+
+        # Don't run pre-reqs
+        os.environ['NO_PREREQ_INSTALL'] = 'true'
```

Comment:
Is this necessary in this environment? I don't think we're using prereqs in this repo are we?